ui/tests/unit/views/AutogenView.spec.js

```diff
@@ -2847,7 +2832,7 @@ describe('Views > AutogenView.vue', () => {
         }
 
         mockAxios.mockResolvedValue(mockData)
-        spyConsole.log = jest.spyOn(console, 'log').mockImplementation(() => {})
+        // spyConsole.log = jest.spyOn(console, 'log').mockImplementation(() => {})
```

Comment:
why keep this line in comment every where? we have version control in place.

Reply:
Đó là một thiếu sót. Tôi đã quên xóa nó trong lần cam kết trước. Cảm ơn

Reply:
That is a shortcoming. I forgot to remove it on the previous commit. Thanks